CMakeLists.txt

```diff
@@ -6,10 +6,23 @@ project(mlboard
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+option(BUILD_TESTS "Build tests." ON)
```

Comment:
I think we should update the readme telling about this option. Let me know what you think.